tests/test_valid_edtf.py

```diff
@@ -1,375 +1,272 @@
 from edtf_validate.valid_edtf import (is_valid_interval, is_valid, isLevel0, isLevel1,
                                       isLevel2, conformsLevel0, conformsLevel1, conformsLevel2)
 
+Level0 = [
+    '1985-04-12',
+    '1985-04',
+    '1985',
+    '0000',
+    '1985-04-12T23:20:30',
+    '1985-04-12T23:20:30Z',
+    '1985-04-12T23:20:30-04',
+    '1985-04-12T23:20:30+04:30',
+    '1964/2008',
+    '2004-06/2006-08',
+    '2004-02-01/2005-02-08',
+    '2004-02-01/2005-02',
+    '2004-02-01/2005',
+    '2005/2006-02',
+]
+
+Level1 = [
+    'Y170000002',
+    'Y-170000002',
+    '2001-21',
+    '-2004-23',
+    '-2010',
+    '1984?',
+    '2004-06~',
+    '2004-06-11%',
+    '-2004~',
+    '-2004-06?',
+    '-2004-06-11%',
+    '201X',
+    '2004-XX',
+    '1985-04-XX',
+    '1985-XX-XX',
+    '-20XX',
+    '-2004-XX',
+    '-1985-04-XX',
+    '-1985-XX-XX',
+    '1985-04-12/',
+    '1985-04/',
+    '1985/',
+    '/1985-04-12',
+    '/1985-04',
+    '/1985',
+    '1985-04-12/..',
+    '1985-04/..',
+    '1985/..',
+    '../1985-04-12',
+    '../1985-04',
+    '../1985',
+    '-1985-04-12/',
+    '-1985-04/..',
+    '-1985/',
+    '-1985-04-12T23:20:30',
+    '-1985-04-12T23:20:30Z',
+    '-1985-04-12T23:20:30-04',
+    '-1985-04-12T23:20:30+04:30',
+    '-1964/2008',
+    '-2004-06/2006-08',
+    '-2004-02-01/2005-02-08',
+    '-2004-02-01/2005-02',
+    '-2004-02-01/2005',
+    '-2005/2006-02',
+]
+
+Level2 = [
+    'Y-17E7',
+    'Y17E8',
+    '1950S2',
+    'Y171010000S3',
+    'Y3388E2S3',
+    '-1859S5',
+    'Y-171010000S2',
+    'Y-3388E2S3',
+    '2001-25',
+    '2001-26',
+    '2001-27',
+    '2001-28',
+    '2001-29',
+    '2001-30',
+    '2001-31',
+    '2001-32',
+    '2001-33',
+    '-2001-34',
+    '-2001-35',
+    '-2001-36',
+    '-2001-37',
+    '-2001-38',
+    '-2001-39',
+    '-2001-40',
+    '-2001-41',
+    '[-1667,1668,1670..1672]',
+    '[..1760-12-03]',
+    '[1760-12..]',
+    '[1760-01,-1760-02,1760-12..]',
+    '[-1667,1760-12]',
+    '[..1984]',
+    '{-1667,1668,1670..1672}',
+    '{1960,-1961-12}',
+    '{..1984}',
+    '2004-06~-11',
+    '2004?-06-11',
+    '?2004-06-~11',
+    '2004-06-%11',
+    '2004-%06-11',
+    '-2004-06?-11',
+    '-2004~-06-11',
+    '?-2004-06-%11',
+    '-2004-06-~11',
+    '-2004-?06-11',
+    '156X-12-25',
+    '15XX-12-25',
+    'XXXX-12-XX',
+    '1XXX-XX',
+    '1XXX-12',
+    '1984-1X',
+    '-156X-12-25',
+    '-15XX-12-25',
+    '-XXXX-12-XX',
+    '-1XXX-XX',
+    '-1XXX-12',
+    '-1984-1X',
+    '2004-06-~01/2004-06-~20',
+    '2004-06-~01/2004-06-~20',
+    '-2004-06-?01/2006-06-~20',
+    '-2005-06-%01/2006-06-~20',
+]
+
+invalid_edtf_dates = [
+    '1863- 03-29',
+    ' 1863-03-29',
+    '1863-03 -29',
+    '1863-03- 29',
+    '1863-03-29 ',
+    '18 63-03-29',
+    '1863-0 3-29',
+    '1960-06-31',
+    '20067890%',
+    'Y2006',
+    '-0000',
+    'Y20067890-14-10%',
+    '20067890%',
+    '+2006%',
+    'NONE/',
+    '2000/12-12',
+]
+
+invalid_edtf_interval = [
```

Comment:
Shall we pluralize this?
```suggestion
invalid_edtf_intervals = [
```